src/components/user/Details.tsx

```diff
@@ -0,0 +1,34 @@
+import { FunctionComponent } from 'react'
+import { useUserContext } from '../../context/user-info'
+import styles from './Details.module.scss'
+
+const UserDetails: FunctionComponent = () => {
+  const user = useUserContext()
+
+  // Nothing to show if not logged in
+  if (!user.info) {
+    return <></>
+  }
+
+  const {
+    github_avatar,
+    github_login,
+    displayname,
+    "dev-flatpaks": flatpaks
+  } = user.info
+
+  return (
+    <div className={styles.details}>
+      <img
+        src={github_avatar}
+        className={styles.avatar}
+      />
+      <div className={styles.textDetails}>
+        <h2>{displayname}</h2>
+        <p>GitHub Account: <a href={`https://github.com/${github_login}`}>@{github_login}</a></p>
+      </div>
+    </div>
```

Comment:
While this will work for now, if a user logged in via twitter, or email, or some other scheme, they won't have github details, so we may want to revisit this as we add future login methods.  perhaps have a TODO to that effect here?

Reply:
Yeah, because this is driven by the data returned from `/userinfo` and currently that just includes github specific keys (i.e. there's no generic array-like structure for the linked accounts) I couldn't make the component very robust to future additions.

Good shout though, should include a code comment to that effect.

Reply:
Thanks